README.md

```diff
@@ -65,6 +65,7 @@ The process to build `react.js` is built entirely on top of node.js, using many
 #### Prerequisites
 
 * You have `node` installed at v4.0.0+ and `npm` at v2.0.0+.
+* You have `gcc` installed or are comfortable installing a compiler if needed. Some of our `npm` dependencies may require a compliation step. 
```

Comment:
Probably worth putting `sudo apt-get install g++` in parenthesis, or linking to the github issue where people discuss how to install gcc for React, just to make life easier for people.  The `apt-get` probably covers the 95% use case.